package.json

```diff
@@ -66,7 +66,7 @@
     "docker:compose:jobs:up": "completed_sqs_queue_url=$(npx ts-node jobs/tests/getContactCompleteQueueUrl.ts) docker-compose -p hrm -f jobs/docker-compose.yml up -d",
     "docker:compose:localstack:up": "docker-compose -p hrm up -d",
     "docker:compose:service:up": "docker-compose -p hrm -f hrm-service/docker-compose.yml up -d",
-    "lint": "eslint --ext ts .",
+    "lint": "cross-env NODE_OPTIONS=--max-old-space-size=8096 eslint --ext ts .",
```

Comment:
ESlint memory usage went through the roof for some reason with this change. We probably don't want to merge this until I've tried to figure out what is going on with that.